### PR TITLE
ci: Move windows to build bdist_wheel

### PR DESCRIPTION
**Pull request by seemethere**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #65587

Also moves the miniconda install to only happen if / when Miniconda3 is
not already installed

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: .jenkins/pytorch/win-test-helpers/setup_pytorch_env.bat [hunk 1]
```diff
@@ -63,26 +64,23 @@ set NUMBAPRO_NVVM=%CUDA_PATH%\nvvm\bin\nvvm64_32_0.dll
...
-    popd
-) else (
-    xcopy /s %CONDA_PARENT_DIR%\Miniconda3\Lib\site-packages\torch %TMP_DIR_WIN%\build\torch\
+:: Install pre-built bdist_wheel
+FOR %%A IN (%PYTORCH_FINAL_PACKAGE_DIR%\*.whl) DO (
+    call python -m pip install %%A
+    if %errorlevel% neq 0 ( exit /b %errorlevel% )
 )
+@echo off
+for /f %%W in ('python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"') DO set site_packages=%%w
```
Comment (seemethere) [inline .jenkins/pytorch/win-test-helpers/setup_pytorch_env.bat:73]: This is setting the `site_packages` variable, don't ask me how it does this in batch, I don't really know

**Comment by seemethere** [comment]
Looks like there's still a lot of unwinding to do with this PR and the fact that we also build test binaries as part of the normal build workflow, going to mark this as a draft for now

**Comment by seemethere** [comment]
Removing the `msft-collab` label since this is not a PR from MSFT

**Comment by malfet** [comment]
Shouldn't this one be closed? (Or at least removed from the project?)

**Comment by seemethere** [comment]
I agree, this one is not a high priority for us at this time and I probably won't be revisiting this PR in the near future.

**Pull request closed without merging**